specs/dotsigstore_bundle/pb/bundle.proto

```diff
@@ -0,0 +1,187 @@
+syntax = "proto3";
+package sigstore;
+
+// https://raw.githubusercontent.com/secure-systems-lab/dsse/9c813476bd36de70a5738c72e784f123ecea16af/envelope.proto
+import "envelope.proto";
+
+option go_package = "github.com/sigstore/xxx/bundle";
+option java_package = "com.github.sigstore.xxx.bundle";
+
+// Notes on versioning.
+// The primary message ('Bundle') MUST be versioned, by populating the
+// 'media_type' field. Semver-ish (only major/minor versions) scheme MUST
+// be used. The current version as specified by this file is:
+// application/vnd.dev.sigstore.bundle.v0.1+json
+// The semantic version is thus 'v0.1'.
```

Comment:
How about moving the version from the subtype into the parameters field? I.e like this: `application/vnd.dev.sigstore.bundle+json; version=0.1`. IMHO this is a cleaner approach, and easier to work with.
Note that the [Sigstore signature spec](https://github.com/sigstore/cosign/blob/main/specs/SIGNATURE_SPEC.md) relies on OCI formats, and so uses a format where the version is part of the subtype: https://github.com/opencontainers/image-spec/blob/main/media-types.md#compatibility-matrix

WDYT @haydentherapper @znewman01 ?

Reply:
i'm fine with this! does this impact existing OCI annotations?

Reply:
Looking around in here: https://github.com/opencontainers/image-spec/blob/main/manifest.md (and `descriptor.md` and others) the `mediaType` is defined as a `string`, `descriptor.md` mandates the media type to be compatible with [RFC6838](https://www.rfc-editor.org/rfc/rfc6838). The suggested media type with version as a parameter is valid according to that RFC. 

So I understand that it should be possible to put this into a OCI registry if desired without violating any spec. The only thing that looks suspicious is that there is a white space in the media type, but it's valid.